odoo_updates/utils.py

```diff
@@ -137,8 +137,7 @@ def __enter__(self):
 
 
 def send_message(message, queue_name):
-    session = boto3.Session()
-    sqs_client = session.client('sqs')
-    url = sqs_client.get_queue_url(QueueName=queue_name)['QueueUrl']
-    response = sqs_client.send_message(QueueUrl=url, MessageBody=message, DelaySeconds=0)
+    resource = boto3.resource('sqs', region_name='us-east-1')
```

Comment:
@Angelfentanez the region shouldn't be here, It must be gotten from [env vars](http://boto3.readthedocs.io/en/latest/guide/configuration.html#environment-variable-configuration) or any other of the [methods](http://boto3.readthedocs.io/en/latest/guide/configuration.html#configuration-file) supported by boto3